app/gui/view/src/project/project_view_top_bar/window_control_buttons.rs

```diff
@@ -0,0 +1,199 @@
+//! The component with buttons in the top left corner. See [[View]].
+
+use ensogl::display::shape::*;
+use ensogl::prelude::*;
+
+use enso_frp as frp;
+use ensogl::application;
+use ensogl::application::Application;
+use ensogl::display;
+use ensogl::display::object::ObjectOps;
+use ensogl::shape;
+use ensogl_hardcoded_theme::application::window_control_buttons as theme;
+
+
+// ==============
+// === Export ===
+// ==============
+
+pub mod close;
+pub mod fullscreen;
+
+
+
+// ==============
+// === Shapes ===
+// ==============
+
+mod shape {
+    use super::*;
+    shape! {
+        alignment = center;
+        (style: Style) {
+            Plane().fill(INVISIBLE_HOVER_COLOR).into()
+        }
+    }
+}
+
```

Comment:
Is this new or old code? In either case, we should use the box shape here to lower the amount of draw calls.